skipcpio/skipcpio.c

```diff
@@ -23,16 +23,65 @@
 #define _GNU_SOURCE
 #endif
```

Comment:
It'd probably make sense to add the libcpio copyright too, e.g.:
```

/*
 * Some code from SEl4 libcpio, which is:
 * Copyright 2017, Data61, CSIRO (ABN 41 687 119 230)
 *
 * SPDX-License-Identifier: BSD-2-Clause
 */
```

Reply:
removed the copied code